File: app/dbmodels/LinkedAsset.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'LinkedAsset',
    'menu'          => ''
);

$table['fields'] = array(
    'id'               => DBModels::type('increments', 
                                         array('visible' => false)),
    'asset_id_1'       => DBModels::type('integer'),
    'asset_id_2'       => DBModels::type('integer'),
);

File: fusionglpi/database/linked_asset.py
from fusionglpi import db


class LinkedAsset(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    asset_id_1 = db.Column(db.Integer, nullable=False, server_default="0")
    asset_id_2 = db.Column(db.Integer, nullable=False, server_default="0")
